### ISSUE TITLE
Error 500 when logging in: user must be a string.

### ISSUE THREAD

**Issue by declan-wade** [nuxt-modules/kinde#39]
Just following the documentation (here)[https://kinde.com/docs/developer-tools/nuxt-module/] to implement a simple authentication page. When logging in, upon loading the `http://localhost:3001/api/callback?code=xxx.xxx&scope=openid+profile+email+offline&state=xxx` URL, I get 500 Error "user must be a string."

```
[nuxt] [request error] [unhandled] [500] user must be a string.
  at Object.setSessionItem (C:\Users\Declan\my-web-app\node_modules\@nuxtjs\kinde\dist\runtime\server\middleware\kinde.mjs:34:17)
  at /C:/Users/Declan/my-web-app/node_modules/@kinde-oss/kinde-typescript-sdk/dist/sdk/utilities/token-utils.js:59:53
  at step (/C:/Users/Declan/my-web-app/node_modules/@kinde-oss/kinde-typescript-sdk/dist/sdk/utilities/token-utils.js:32:23)
  at Object.next (/C:/Users/Declan/my-web-app/node_modules/@kinde-oss/kinde-typescript-sdk/dist/sdk/utilities/token-utils.js:13:53)
  at /C:/Users/Declan/my-web-app/node_modules/@kinde-oss/kinde-typescript-sdk/dist/sdk/utilities/token-utils.js:7:71
  at new Promise (<anonymous>)
  at __awaiter (/C:/Users/Declan/my-web-app/node_modules/@kinde-oss/kinde-typescript-sdk/dist/sdk/utilities/token-utils.js:3:12)
  at commitUserToMemoryFromToken (/C:/Users/Declan/my-web-app/node_modules/@kinde-oss/kinde-typescript-sdk/dist/sdk/utilities/token-utils.js:45:79)
  at /C:/Users/Declan/my-web-app/node_modules/@kinde-oss/kinde-typescript-sdk/dist/sdk/utilities/token-utils.js:79:38
  at step (/C:/Users/Declan/my-web-app/node_modules/@kinde-oss/kinde-typescript-sdk/dist/sdk/utilities/token-utils.js:32:23)
[Vue Router warn]: No match found for location with path "/api/callback?code=FjCO4AtQy-xxx"
[Vue Router warn]: No match found for location with path "/api/callback?code=FjCO4AtQy-xxx
[Vue Router warn]: No match found for location with path "/api/callback?code=FjCO4AtQy-xxx"
```
Not sure what is going on, it's an extremely simple implementation using minimal code.

Version details:
```
Windows 11 Pro
NPM: v9.6.7
Node: v18.17.0
Nuxt 3.9.0 with Nitro 2.8.1
vue-router: "^4.2.5"
Edge: Version 120.0.2210.91 (Official build) (64-bit)
```

**Comment by DanielRivers** [comment]
@declan-wade This is likely to be a regression in 0.1.5, could you downgrade to 0.1.4 and try please?

**Comment by declan-wade** [comment]
Huh - yes, you are right. Changing to 0.1.4 with no other changes, fixed the issue. Can we leave this issue open so the root cause can be investigated and fixed in a future release?
Thanks for the assistance :)

**Comment by DanielRivers** [comment]
For sure, I will look at a solution ASAP.  Thanks for raising.

**Comment by DanielRivers** [comment]
@declan-wade This has been fixed in 0.1.6 just released.

**Comment by declan-wade** [comment]
Can confirm bumping to 0.1.6 fixed the issue - great work!
